odoo100/scripts/10.0-full_requirements.txt

```diff
@@ -29,5 +29,3 @@ sphinx-patchqueue==0.4.0
 sphinx==1.2.3
 unidecode
 git+https://github.com/savoirfairelinux/num2words.git@0.5.4
```

Comment:
@moylop260 num2words? I'm not able to find it, can you point me to the line where it is repeated please?